tabout/src/lib.rs

```diff
@@ -94,7 +94,7 @@ pub fn tabulate_output<S: std::string::ToString, W: std::io::Write>(
 
     for row in &display_rows {
         for (idx, col) in row.iter().enumerate() {
-            let max_width = col_widths.get(idx).cloned().unwrap_or_else(|| col.len());
+            let max_width = col_widths.get(idx).cloned().unwrap_or(col.len());
```

Comment:
Note that for these, we're now unconditionally calling `col.len()`.
Previously, it would only be called in the fallback case.
I don't think this matters in this file, but in general, that eager computation is undesirable.

Reply:
As far as I can tell `col` is a `String` thus `col.len()` is just reading a property and cheaper than calling a function conditionally.

Will keep in mind in the future though.